_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,304 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable functionality that can be operated by device motion.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has one or more registered [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation 1
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available to disable the event.
+
+## Expectation 2
+
+The [instrument][] is [visible][].
+
+## Expectation 3
+
+The [instrument][] is [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
```

Comment:
These two assumptions can certainly be merged together, or event with the first one.
I have no strong feeling between keeping them separate or not. Our current practice tend to be to have as few expectations as needed, but that is not necessarily the best way to do…

Reply:
In the previous version of the rule, it was just one expectation. But I always feel that it is more readable like this. If someone really wants this to be just one expectation, I can change it back.

Reply:
Thinking about it, I do like having separate expectations as it's easier to read and allow to pinpoint the problem more precisely (better for reporting error and indicating fixes).